### ISSUE TITLE
Buyable Ores

### ISSUE THREAD

**Issue by DragoWhooves** [ColonyPlusPlus/ColonyPlusPlus-mod#36]
Please make Ores buyable in the shop

something like

Iron - 30 coins for 10 Ore
Coal - 25 coins for 10 Ore
Gypsum - 20 coins for 10
Gold - 40 coins for 10 Ore

**Comment by tommykent1210** [comment]
Yes, you should definitely be able to buy at least the ore (and probably the finished iron/gold ingots) for a cost. The main thing to bare in mind is: how much does it cost (in coins, and thus gold ingots) to buy a gold ingot. It has to be diminishing returns really.

Could be done as part of #38

**Comment by DragoWhooves** [comment]
well currently 3 gold ore makes 1 coin, but if Minting is rebalanced (see #37 ) then the values would be different, it should be that buying the gold ore or ingots should end you up with less

**Comment by Dorky106** [comment]
its a 2:2 ratio currently

**Comment by tommykent1210** [comment]
Yes, basically I'd say the economy will be more important once you can sell items, keeping that balanced will be hard
